### ISSUE TITLE
DD version tag improvements for edxapp

### ISSUE THREAD

**Issue by robrap** [edx/edx-arch-experiments#751]
We've had issues where GoCD has new builds (possibly due to config or other changes), and this can't be seen in the DD version tag. The version tag just uses the repo hash. Do we want to use the GoCD id instead, so we can see the actual deployment?

- This doesn't happen often, but when it does happen, it does add confusion. Without this information it looks like a deployment didn't happen. If you know it did happened, you can't take advantage of the DD deployments functionality to see if it is related to errors, etc.
- Would we want the same fix for all services, or just start with edxapp?
- This should probably be blocked on containerization, in case that affects how we'd implement pipeline changes to make this work.

**Comment by robrap** [comment]
This duplicates:
- https://github.com/edx/edx-arch-experiments/issues/670.

**Comment by robrap** [comment]
Closing in favor of https://github.com/edx/edx-arch-experiments/issues/670.
